modApiServer/src/org/aion/api/server/rpc/ApiWeb3Aion.java

```diff
@@ -2872,6 +2838,14 @@ private IRepository getRepoByJsonBlockId(String _bnOrId) {
         return ac.getRepository().getSnapshotTo(b.getStateRoot());
     }
 
+    private AionBlock getBlockByBN(Long bn) {
```

Comment:
I think long is better than Long here (probably doesn't make sense to pass in null)